index.js

```diff
@@ -1,17 +1,42 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.length += 1;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (this.items[pos] > 0 && this.items[pos] <= this.items.length) {
```

Comment:
Aqui no if, vc precisa conferir se a **posição** existe no array.
O que vc está fazendo é conferindo se o **valor** do elemento é maior que zero ou menor q o comprimento, então só precisaria ser
`if ( pos > 0 && pos < this.items.length ) {...}`

Reply:
Vlw DK!